backend/base/models.py

```diff
@@ -1,3 +1,54 @@
 from django.db import models
-
+from django.contrib.auth.models import User
 # Create your models here.
+
+
+class Product(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    image = models.ImageField()
+    price = models.DecimalField(max_digits=7, decimal_places=2)
+    count_in_stock = models.IntegerField(default=0)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Order(models.Model):
+    user = models.ForeignKey(User,on_delete=models.SET_NULL, null=True)
+    payment_method = models.CharField(max_length=255)
+    shipping_price = models.DecimalField(max_digits=7, decimal_places=2)
+    total_price = models.DecimalField(max_digits=7, decimal_places=2)
+    is_paid = models.BooleanField(default=False)
+    paid_at = models.DateTimeField(auto_now_add=False, null=True, blank=True)
+    is_delivered = models.BooleanField(default=False)
+    delivered_at = models.DateTimeField(auto_now_add=False, null=True, blank=True)
```

Comment:
should this be auto added? this should be null until it is delivered right? so it should be manually added

Reply:
You are right. This should be manually added once paid/ delivered.